package.json

```diff
@@ -8,7 +8,7 @@
     "build": "babel lib -d dist --presets es2015,stage-0",
     "eslint": "eslint lib/",
     "serve": "node dist/index.js",
-    "test": "mocha lib/ --compilers js:babel-register --require babel-polyfill -R mocha-trumpet-reporter",
+    "test": "mocha lib/ --compilers js:babel-register --require babel-polyfill -R nyan",
```

Comment:
Hah! I should have checked the authors before replacing it with:
![nyan](http://i.giphy.com/TBLVbGSqA7e24.gif)